### ISSUE TITLE
Unable to hide warnings in a mixed .NET Core and .Net Framework solution

### ISSUE THREAD

**Issue by landerah** [BMuuN/vsts-assemblyinfo-task#136]
We are building a solution that includes both .NET Core and .Net Framework projects.
With the yaml below:
```
- task: Assembly-Info-NetFramework@2
  displayName: 'Set Assembly Manifest Data for .Net Framework Assemblies'
  inputs:
    Path: '$(Build.SourcesDirectory)'
    FileNames: '**\AssemblyInfo.cs'
    InsertAttributes: true
    FileEncoding: 'auto'
    VersionNumber: '$(assemblyVersion)'
    FileVersionNumber: '$(assemblyVersion)'
    InformationalVersion: '$(Build.BuildNumber)'

- task: Assembly-Info-NetCore@2
  displayName: 'Set Assembly Manifest Data for .Net Core and Standard Assemblies'
  inputs:
    Path: '$(Build.SourcesDirectory)'
    FileNames: '**/*.csproj'
    InsertAttributes: true
    FileEncoding: 'auto'
    VersionNumber: '$(assemblyVersion)'
    FileVersionNumber: '$(assemblyVersion)'
    InformationalVersion: '$(Build.BuildNumber)'
```
We end up with a number of warnings relating to Assembly-Info-NetCore@2 finding projects that are no .Net Core / Standard. Is there a way to switch these off or handle a mixed environment better? It looks like there is no way to prevent these warnings.

**Comment by hermanvos** [comment]
We have the same issue, and with a large project there is a bunch of them that are generated into our logs. Hopefully the PR helps.

**Comment by BMuuN** [comment]
Fixed with v2.2.103

**Comment by hermanvos** [comment]
Thanks heaps this works brilliantly now just tested in our CI builds
